drivers/tty/serdev/pibridge_trace.h

```diff
@@ -13,7 +13,17 @@
 #define PIBRIDGE_TRACE_MAX_FIFO_DUMP		128
 #define PIBRIDGE_TRACE_MAX_PACKET_DATA_LEN	256
 
-/*  Header */
+/*
+ * pibridge_gate_header_class
+ *
+ * Print the elements of a gateway header.
+ *
+ * dst: destination address. Specifies to which RevPi device the packet is sent.
+ * src: source address of the sending device.
+ * cmd: command this header belongs to.
```

Comment:
I can't really make sense of "command this header belongs to". How can a header belong to a command?
Maybe something like "command issued by this header"?

Somewhat tangential, I realize only now that the command is emitted as a decimal value. However the definition in <a href="https://github.com/RevolutionPi/piControl/blob/master/ModGateRS485.h#L48">ModGateRS485.h</a> uses hexadecimal values. So it's not possible to directly correlate printed values with the header file. On the other hand, an even more user-friendly approach would be to emit the command as a string. E.g., an array of strings could be defined and then the trace event would just look up the string in the array and print it.